ragna/deploy/_ui/api_wrapper.py

```diff
@@ -81,13 +87,16 @@ def upload_endpoints(self):
         }
 
     async def start_and_prepare(
-        self, name, documents, source_storage, assistant, params
+        self, name, documents, corpus_name, source_storage, assistant, params
     ):
+        if corpus_name is None:
```

Comment:
Not sure what name should be passed here...

Reply:
When a user uploads files, they have the option to pass a Corpus Name. If they do not pass a Corpus Name, then we hit this.

Reply:
Shouldn't we put the default corpus name as a default value in that text field instead of quietly resolving it in the background? Meaning, instead of allowing users to not fill the text field, just have `"default"` there as value. Functionally this should be the same, but the user knows explicitly what corpus the documents were uploaded to.